cmd/kubeadm/app/cmd/init.go

```diff
@@ -114,12 +117,18 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 	var featureGatesString string
 	var criSocket string
 	var ignorePreflightErrors []string
+        var verbosityLevel int
 
 	cmd := &cobra.Command{
 		Use:   "init",
 		Short: "Run this command in order to set up the Kubernetes master.",
 		Run: func(cmd *cobra.Command, args []string) {
 			var err error
+
+                        // Required for verbose messages to be printed
+                        goflag.Set("logtostderr", "true")
+                        goflag.Set("v", strconv.Itoa(verbosityLevel))
```

Comment:
this probably will be not needed once glog flags properly imported into pflag.

Reply:
Corrected.